public/apps/configuration/_index.scss

```diff
@@ -14,7 +14,7 @@
  */
 
 .panel-header-count {
-  color: #687078;
+  color: $euiTextSubduedColor;
```

Comment:
Before:

<img width="1788" alt="Screenshot 2023-08-18 at 1 47 52 PM" src="https://github.com/opensearch-project/security-dashboards-plugin/assets/17432265/9b534006-cb8f-4452-8649-dc006516e856">

After:

<img width="1789" alt="Screenshot 2023-08-18 at 1 52 03 PM" src="https://github.com/opensearch-project/security-dashboards-plugin/assets/17432265/4102bb4e-2a4b-4316-90be-a540c9344235">

This affects the number in paren next to the config type. In this case `Roles **(42)**`.